python/ray/train/BUILD

```diff
@@ -412,31 +412,31 @@ py_test(
     name = "test_lightning_checkpoint",
     size = "medium",
     srcs = ["tests/test_lightning_checkpoint.py"],
-    tags = ["team:ml", "exclusive", "ray_air", "gpu"],
+    tags = ["team:ml", "exclusive", "ray_air", "gpu", "ptl_v2"],
```

Comment:
So, these tests will run in both the AIR unit tests pipeline w/ lightning v1, as well as this new Lightning v2 pipeline?

Does the old one with lightning v1 still work with your changes to the tests?

Reply:
Yes it still works: [here](https://buildkite.com/ray-project/oss-ci-build-pr/builds/19278#01879d5c-0eb7-4a5d-8cc1-cbf19a457e7b)